peer/peer_test.go

```diff
@@ -40,6 +40,11 @@ type conn struct {
 	proxy bool
 }
 
+const (
+	addr1 = "10.0.0.1:16111"
```

Comment:
Please use more meaning names, like outPeerAddr or something like this

Reply:
It really depends on the test.
The addresses have no meaning, just something that you need to specify to make things work.